client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/index.tsx

```diff
@@ -33,10 +41,20 @@ const SiteMigrationCredentials: Step = function ( { navigation } ) {
 				formattedHeader={
 					<FormattedHeader
 						id="site-migration-credentials-header"
-						headerText={ translate( 'Tell us about your site' ) }
-						subHeaderText={ translate(
-							'Please share the following details to access your site and start your migration.'
-						) }
+						headerText={
+							isEnglishLocale
+								? translate( 'Tell us about your WordPress site' )
+								: translate( 'Tell us about your site' )
+						}
+						subHeaderText={
+							isEnglishLocale
+								? translate(
+										'Please share the following details to access your site and start your migration to WordPress.com.'
```

Comment:
The text in the image is "Drop your details below and we'll get started with your migration to WordPress.com.", but your text also looks good. Have we decided to go with this one instead?